src/index.html

```diff
@@ -7,6 +7,40 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="content">
+      <div class="content__item calendar calendar--start-day--sun">
```

Comment:
Add a modifier month-length for the calendar...
 div class="...calendar---month-length---29..."